app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -755,8 +755,13 @@ class BrowserFragment : WebFragment(), LifecycleObserver, View.OnClickListener,
             Settings.getInstance(requireContext()).shouldUseBiometrics() &&
             Biometrics.hasFingerprintHardware(requireContext())
```

Comment:
This looks like it's supposed to exactly match another conditional statement found in `onPause`, but there's no clear connection between them.  Suggest breaking them out into a shared function, so we never change one but forget the other.